components/ChapterCard.tsx

```diff
@@ -34,21 +38,23 @@ const ChapterCard = () => {
         <div className='flex group-hover:invisible w-full justify-between text-grey'>
           <div className='flex flex-col justify-center items-center gap-2'>
             <span className='self-end text-[14px] leading-[17px] font-medium'>
-              Values, Types, and Operators
+              {exercises}
             </span>
           </div>
         </div>
 
-        <div className='chapter-card__btn-container'>
+        <Link href={link} className='chapter-card__btn-container'>
           <CustomButton
             title='View More'
             containerStyles='w-full py-[16px] rounded-full bg-[#B2980B]'
             textStyles='text-white text-[14px] leading-[17px] font-bold'
             rightIcon='/right-arrow.svg'
-            handleClick={() => setIsOpen(true)}
           />
-        </div>
+        </Link>
       </div>
+
+      <Link href={link}></Link>
+      {/* <CardDetails isOpen={isOpen} closeModal={() => setIsOpen(false)} /> */}
```

Comment:
Commented out code?

Reply:
I was doing test to make the link work, I removed the comment and cleaned the component